Digipost.Signature.Api.Client.Core/Extensions/EnumExtensions.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
 using Digipost.Signature.Api.Client.Core.Enums;
+using Schemas;
```

Comment:
Takk gud for at det ble litt annen struktur på genererte xsd-klasser sånn at vi fikk en slik som denne i alle filer de brukes ...